styleguide/derek/pattern-components/headers/headers.js

```diff
@@ -46,10 +46,10 @@ function svgBannerStringSplitter(str, l){
 /**
  * Set's the svg content.
  */
-function svgBannerSetContent() {
+function svgBannerSetContent(primaryBox = 24) {
```

Comment:
I don't think we want to change the default value here.  It could have unexpected results.

Reply:
To clarify, the default was 25 before, and this changes it to 24.  We should leave the default at 25.

Reply:
There is an issue on D7 and I'm sure D8 where the text butts up to the edge of the white box. Changing it to 24 resolved that issue.. Looks like this:
![screen shot 2018-05-14 at 11 47 18 am](https://user-images.githubusercontent.com/8450357/40132594-d2f59f84-5902-11e8-9d62-8baa437f6466.png)